hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -74,6 +78,20 @@ public void configure() {
     }
 
     
+    /**
+     * This method performs the steps required before configuring the route
+     * 1. Set the transaction id generator for messages
+     * 2. Load application properties
+     * 3. Initializes the validator classes
+     */
+    private void init() {
```

Comment:
I like that you pulled this code into a separate method since it's not part of the actual route() definition.